Data/RoomRepository.cs

```diff
@@ -55,19 +55,30 @@ public async Task<bool> PutRoom(Room room)
             }
         }
 
-        private bool RoomExists(int id)
+        public async Task<bool> RoomExists(int id)
         {
             return _context.Rooms.Any(e => e.Id == id);
```

Comment:
Two options to get rid of this warning:

1. Replace `async Task<bool>` with `bool`
2. Use `await _context.Rooms.AnyAsync(e => e.Id == id)`